### ISSUE TITLE
Toggling a line from the legend through the keyboard in high contrast mode does not work

### ISSUE THREAD

**Issue by FrankTaubert** [highcharts/highcharts#16562]
#### Expected behaviour
In High contrast mode: It should be possible to un-select a line in a line chart by pressing space bar when the name is selected in the legend next to the chart.

#### Actual behaviour
The line is not hidden. Clicking on the label works.

Looking at the code: it seems the keydown event is register twice and leads to the line being hidden and immediately being shown again.

This happens in `KeyboardNavigation.prototype.init = function (chart, components)`  (accessibility.src.js).
Wher in `Accessibility.prototype.init = function (chart) {`
1) `this.keyboardNavigation = new KeyboardNavigation(chart, this.components);` registers a key down event **(number 1)**
2) the next line is `this.update();` 
3) which calls `setHighContrastTheme` 
4) which triggers an update of the series 
5) which updates of the chart and `updateA11yEnabled`  
7) which leads to the creation of a `new Accessibility(` 
8) which leads to the creation of a new `KeyboardNavigation` 
9) which registers a key down event **(number 2)**
10) it will not register another one, as step 3) `setHighContrastTheme` is not called again, as `highContrastModeActive` is true at this point.

#### Live demo with steps to reproduce
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/line-basic
The issue is visible when I turn on the Windows high contrast mode and open the above jsfiddle in the browser
Steps:
- in high contrast mode
- navigate to "installation" in the legend on the right side of the graph
- press "space"
- "installation" line stays visible / is not deactivated
- click on the same label
- "installation" line disappears / is deactivated

#### Product version
v9.3.0 (2021-10-21)

#### Affected browser(s)
Edge Version 95.0.1020.30 (Official build) (64-bit)

**Comment by raf18seb** [comment]
Hi @FrankTaubert, thanks for reporting the issue and describing it well.

**Comment by FrankTaubert** [comment]
Hi @raf18seb, I wanted to ask if you have an estimate on if you are going to fix this and how long it will take? This was flagged as severity 1 bug by the accessibility team of my organisation. So it's urgent for us, that this gets fixed.

**Comment by raf18seb** [comment]
Hi @FrankTaubert, we'll look into this issue this week and provide more info soon.

**Comment by oysteinmoseng** [comment]
@FrankTaubert A PR with the fix is under review, and you can expect this issue to be resolved in the next Highcharts release. Thank you for providing such great detail and investigative work in the bug report!

**Comment by FrankTaubert** [comment]
Hi @oysteinmoseng, is there a target date to that release?

**Comment by oysteinmoseng** [comment]
Hi @FrankTaubert, we do not set specific target dates, but you can expect a new release within the next few weeks. If you wish, you could also try the pre-release [nightly build of Highcharts](https://www.highcharts.com/docs/getting-started/install-from-npm#installing-nightly-builds-of-highcharts). If this is not possible for you, let me know and I'll see if we can speed up the release.

**Comment by FrankTaubert** [comment]
Hi @oysteinmoseng, I would of course prefer a proper release, but it think I can make a nightly build work 😆 

I looked at the link and tried it. Looking at the code taht I got, it does not seem to include the fix. Also looking at the nightly branch https://github.com/highcharts/highcharts-dist/tree/nightly, the last commit on there is 18 days old, whereas the fix is only 10 days old. This probably just needs a bump?

**Comment by oysteinmoseng** [comment]
Apologies for the inconvenience, it seems there are some issues in the master branch that causes automated tests to fail.

I will see if we can speed up the release.

If you happen to be able to download the latest files from github.highcharts.com (e.g. [https://github.highcharts.com/modules/accessibility.js](https://github.highcharts.com/modules/accessibility.js)) and host them locally you could try that in the meantime.

**Comment by FrankTaubert** [comment]
Hi @oysteinmoseng, just wanted to let you know that this issue was fixed for us with the new version. Thanks for pushing that through so fast. Really appreciated.

**Comment by oysteinmoseng** [comment]
Great to hear!
